package.json

```diff
@@ -24,6 +24,7 @@
   },
   "devDependencies": {
     "grunt": "~0.4.5",
+    "grunt-cli": "^1.3.1",
```

Comment:
What's this for?

Reply:
It's for using Grunt. This project doesn't work "as-is" and needed this dependency added. CI was failing until https://github.com/dequelabs/grunt-axe-webdriver/pull/40/commits/f9cc4146b5905f8e267790498c6c722aac340197.

Reply:
CI still fails, so unsure if above fixes it.

Reply:
Adding this dependency is 100% necessary. The `grunt` program won't exist otherwise.